### DISCUSSION TITLE
Welcome to playwright-typescipt-playwright-test Discussions!

### DISCUSSION THREAD

**Discussion by akshayp7** [akshayp7/playwright-typescript-playwright-test#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by krishnamoorthi1207** [comment]
<angular2-multiselect [data]="dropdownList" [(ngModel)]="selectedItems" 
    [settings]="dropdownSettings" 
    (onSelect)="onItemSelect($event)" 
    (onDeSelect)="OnItemDeSelect($event)"
    (onSelectAll)="onSelectAll($event)"
    (onDeSelectAll)="onDeSelectAll($event)">
</angular2-multiselect> when i try to access angular multiselect dom element from playwright frame work not working. normal select is working. can u any one help this

**Comment by akshayp7** [comment]
Hi Thank you for using my framework and thanks for your feedback. 
Can you please  try first clicking on select box the identifying the select by text and then clicking the option.
Means instead of using select directly you can click on the select box then identify element and then click
